src/v2/ion/i18nTransformer.js

```diff
@@ -144,6 +144,8 @@ const I18N_OVERRIDE_MAPPINGS = {
 
   'enroll-profile.userProfile.lastName': 'oie.user.profile.lastname',
   'enroll-profile.userProfile.firstName': 'oie.user.profile.firstname',
+  'enroll-profile.userProfile.countryCode': 'oie.user.profile.countryCode',
```

Comment:
This was an english leak that got introduced with https://github.com/okta/okta-signin-widget/commit/9b6cc765144fe8a549bcd12a0530840e6ed140c8

Before: 

![Screen Shot 2021-12-15 at 9 40 50 AM](https://user-images.githubusercontent.com/23267876/146240010-46c3fe31-ea5d-43ba-a868-8d0210ed93e7.png)




After: 
![Screen Shot 2021-12-15 at 9 52 49 AM](https://user-images.githubusercontent.com/23267876/146240033-0fbd2c77-4a6e-4330-aa6e-8faa6700b41b.png)

Reply:
LGTM. Thanks for the fix!

Reply:
I still had to ignore this mock file in the test because it was flagging the dropdown values as english leaks. The JIRA is linked in that comment